Cambalab/ns-vue-radio
# Provide base icons to be able to generate icons and splashes

This PR provides:
- Base icons and splashes to be able to generate new ones

How to test:
- run `tns resources generate icons /path/to/an/img`
- visit one of the asset folders

Expected result: icons should change to the new image


Closes #148
